.github/workflows/pre_commit_checks.yml

```diff
@@ -6,22 +6,22 @@ jobs:
   pre-commit:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+    - uses: actions/checkout@v4
 
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        id: py311
-        with:
-          python-version: 3.11
-          cache: pip
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      id: py311
+      with:
+        python-version: 3.11
+        cache: pip
 
-      - name: Install pip dependencies
-        run: pip install -r requirements-dev.txt
+    - name: Install pre-commit
+      run: pip install pre-commit==3.5.0
```

Comment:
3.6.0 is the latest version.

Reply:
Good catch, this one was just released over the weekend; I modified this file before that. I'll update!